sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy

```diff
@@ -114,33 +89,34 @@ class APISpec extends StorageSpec {
 
         premiumFileServiceClient = setClient(premiumCredential)
         premiumFileServiceAsyncClient = setAsyncClient(premiumCredential)
-
-        // Print out the test name to create breadcrumbs in our test logging in case anything hangs.
-        System.out.printf("========================= %s.%s =========================%n", className, testName)
     }
 
     /**
      * Clean up the test shares, directories and files for the account.
      */
     def cleanup() {
-        interceptorManager.close()
-        if (ENVIRONMENT.testMode != TestMode.PLAYBACK) {
-            def cleanupFileServiceClient = new ShareServiceClientBuilder()
-                .retryOptions(new RequestRetryOptions(RetryPolicyType.FIXED, 3, 60, 1000, 1000, null))
-                .connectionString(connectionString)
-                .buildClient()
-            for (def share : cleanupFileServiceClient.listShares(new ListSharesOptions().setPrefix(methodName.toLowerCase()), null, Context.NONE)) {
-                def shareClient = cleanupFileServiceClient.getShareClient(share.getName())
-
-                if (share.getProperties().getLeaseState() == LeaseStateType.LEASED) {
-                    createLeaseClient(shareClient).breakLeaseWithResponse(new ShareBreakLeaseOptions().setBreakPeriod(Duration.ofSeconds(0)), null, null)
-                }
-
-                shareClient.deleteWithResponse(new ShareDeleteOptions().setDeleteSnapshotsOptions(ShareSnapshotsDeleteOptionType.INCLUDE), null, null)
+        def cleanupFileServiceClient = new ShareServiceClientBuilder()
+            .connectionString(connectionString)
```

Comment:
should we add the retry options?

Reply:
there's exponential retry by default. what was here before was causing 5xx timeouts...